optimize/qa/schema-integrity-tests/pom.xml

```diff
@@ -115,9 +164,9 @@
               <removeVolumes>true</removeVolumes>
               <envVars>
                 <NEW_ES_VERSION>${elasticsearch.test.version}</NEW_ES_VERSION>
-                <NEW_ES_PORT>${new.elasticsearch.port}</NEW_ES_PORT>
+                <NEW_ES_PORT>${new.database.port}</NEW_ES_PORT>
```

Comment:
I am assuming we will be using those env-variables for OS as well once the upgrade is implemented, right? As right now they are only read in the docker-compose file. If my assumption is correct, they should be renamed to `NEW_DB_PORT`/`OLD_DB_PORT`

Reply:
Actually not really. I included this in the README, that I have no intention make changes to allow these to be run locally. The docker profile is just for local testing. I think we can live with OpenSearch not being testable locally given how infrequently we update these tests